etc/st2packgen/files/sensors/sensor_base.py

```diff
@@ -1,10 +1,16 @@
+# based on https://cobe.io/blog/posts/kubernetes-watch-python/
+
 import ast
 import json
 import sys
-
-import requests
-from requests.auth import HTTPBasicAuth
-
+import base64
+import select
+import socket
+import ssl
+import io
+import re
+
+from http_parser.parser import HttpParser
```

Comment:
Likely due to some restriction on the circle container the compiled version of HttpParser isn't available. So my suggestion is to use the following:

```
try:
    from http_parser.parser import HttpParser
except ImportError:
    from http_parser.pyparser import HttpParser
```

Which will allow it to fall back to the pure python parser if the compiled version isn't available.